src/tools/gophish_import.py

```diff
@@ -261,15 +264,14 @@ def main() -> None:
         )
     except ValueError:
         logging.critical(
-            '"{}"is not a valid logging level.  Possible values are debug, info, warning, and error.'.format(
-                log_level
-            )
+            '"%s"is not a valid logging level.  Possible values are debug, info, warning, and error.',
```

Comment:
I think you need a space here.
```suggestion
            '"%s" is not a valid logging level.  Possible values are debug, info, warning, and error.',
```

Reply:
See 825962be74cfc038bdf689b711d0b91860d5c0b5